t/t7113-post-index-change-hook.sh

```diff
@@ -12,6 +12,36 @@ test_expect_success 'setup' '
 	git commit -m "initial"
 '
 
+test_expect_success 'post-index-changed' '
```

Comment:
hopefully this is in a separate commit

Reply:
yeah, the merge target changed since I queued this PR and it looks like that brought in more noise here.